### ISSUE TITLE
Failing with trailing POS in UNIT line

### ISSUE THREAD

**Issue by IanMayo** [debrief/pepys-import#513]
The users are processing their first NISIDA file.

The first line is the `UNIT/` line we need for the date.  But, it looks like the `POS/` from the second line has been pulled onto the first, so it looks like this:
```
UNIT/ADRI/JAN19/SRF/POS/
311002Z/3623.00N/00412.02E/GPS/359/03/-/
311030Z/3653.02N/00430.06E/GPS/356/13/-/
```
But, it should be like this:
```
UNIT/ADRI/JAN19/SRF/
POS/
311002Z/3623.00N/00412.02E/GPS/359/03/-/
311030Z/3653.02N/00430.06E/GPS/356/13/-/
```
I've asked them to remove the `POS/` from the first line. We don't need them to put it back onto the second line, since we don't rely on the `POS/` header.

But, we should modify Pepys so that it robustly handles extra fields in the `UNIT/` line.

**Comment by IanMayo** [comment]
Ok, I've created a PR with a temporary fix.

But, the underlying issue is that we can drop out of importing the `UNIT` line, which leaves `self` in an inconsistent state.  When processing `UNIT` fails, we should terminate the import.

The fragility of NISIDA files means we should also be prepared for other UNIT line errors, including something like a malformed date (with typo in month).

**Comment by robintw** [comment]
Fixed by #514
